heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -591,6 +591,7 @@ export default class JestPlugin implements IHeftTaskPlugin<IJestPluginOptions> {
       rootDir: buildFolderPath,
 
       silent: options.silent || false,
+      verbose: (taskSession.parameters.verbose || taskSession.parameters.debug) && !options.silent,
```

Comment:
Hmm... given that silent can only be specified in the config, and CLI is generally considered to take precedence over existing config, I'd imagine that we should set verbose regardless of the `options.silent` parameter and just consider the `--verbose`/`--debug` parameters

Reply:
fixed